streams/src/test/java/org/apache/kafka/test/StreamsTestUtils.java

```diff
@@ -78,4 +72,24 @@ public static Properties minimalStreamsConfig() {
         return results;
     }
 
+    // TODO: these three static functions are added because some non-TopologyBuilder unit tests need to access the internal topology builder,
+    //       which is usually a bad sign of design patterns between TopologyBuilder and StreamThread. We need to consider getting rid of them later
+    public static InternalTopologyBuilder internalTopologyBuilder(final StreamsBuilder streamsBuilder) throws NoSuchFieldException, IllegalAccessException {
```

Comment:
Here and the next method. Can we wrap them with:
```
try {
.../
}catch(final Exception e) {
  throw new RuntimeException(e);
}
```
We can then get rig of the `throws Exception` from all the tests and from the `KStreamTestDriver`